.github/workflows/cucumber-tests.yml

```diff
@@ -42,29 +37,37 @@ jobs:
       - name: Build Docker Images
         env:
           DOCKER_BUILDKIT: '1'
-          CSO_DEBUG_ENABLED: 'false'
         run: |
           docker-compose build --parallel efiling-demo efiling-frontend efiling-api
 
```

Comment:
This like from a previous commit has cut down some of the time needed if there's a cache miss.